src/xunit.v3.core/CollectionBehavior.cs

```diff
@@ -7,14 +7,16 @@ public enum CollectionBehavior
 {
 	/// <summary>
 	/// By default, generates a collection per assembly, and any test classes that are not
-	/// decorated with <see cref="CollectionAttribute"/> will be placed into the assembly-level
-	/// collection.
+	/// decorated with <see cref="CollectionAttribute"/> or 
+	/// <see cref="CollectionAttribute{TCollectionDefinition}"/> will be placed into the
+	/// assembly-level collection.
 	/// </summary>
 	CollectionPerAssembly,
 
 	/// <summary>
 	/// By default, generates a collection per test class for any test classes that are not
-	/// decorated with <see cref="CollectionAttribute"/>.
+	/// decorated with <see cref="CollectionAttribute"/> or
+	/// <see cref="CollectionAttribute{TCollectionDefinition}"/>.
 	/// </summary>
 	CollectionPerClass
```

Comment:
Reading this reminds me that we probably need to make sure that the following two classes end up in the same collection:

```csharp
public class TestClass1 { ... }

[Collection<TestClass1>]
public class TestClass2 { ... }
```

which means the collection ID for default classes during `CollectionPerClass` also needs to use `UniqueIDGenerator`.

And in thinking about this, it probably needs to be the case that we want the name to continue to include the class name, and just use the unique ID as a discriminator. So today the default collection in `CollectionPerClass` is called something like `"Test collection for <Type.FullName>"`, whereas now both the default and `[Collection<T>]` should probably have a name of `"Test collection for <Type.FullName> (<UniqueID>)"` for the uniqueness benefit.

Reply:
There are tests that might be affected, probably in `CollectionAcceptanceTests`, `CollectionPerClassTestCollectionFactoryTests`, and `Xunit3AcceptanceTests`.

Reply:
Just trying to make sure I understand. In the current situation, if I do this:

`
[CollectionDefinition("bacon")]
public class TestClass1 { ... }

[Collection("bacon")]
public class TestClass2 { ... }
`

Are these classes considered part of the same collection today? If so, then completely agreed. But I wasn't sure what the behavior is right now if a test class is adorned with `CollectionDefinitionAttribute` rather than putting that attribute on a fixture.

Reply:
Actually ignore this for now. I might want to play around with some things to try to decide whether it's actually what I really want or not. It may be one of those ideas that sounds good for 5 minutes, but bad after you spend half a day with it. 😂

Reply:
Gold 5 standing by

Reply:
Resolving as "ignore me" on this one. 😄